docs/topics/Gateway.md

```diff
@@ -487,7 +483,6 @@ The resumed event is dispatched when a client has sent a [resume payload](#DOCS_
 
 | Field   | Type             | Description                                    |
 | ------- | ---------------- | ---------------------------------------------- |
```

Comment:
Probably better to remove the entire table then

Reply:
👍